net/sscep/Makefile

```diff
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sscep
+PKG_VERSION:=0.10.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v${PKG_VERSION}.tar.gz
+PKG_SOURCE_URL:=https://github.com/certnanny/sscep/archive/
+PKG_HASH:=489cc8e093986776eb3f15082bf766778f707176f3cd604bf0ef1008da06b8e5
+
+PKG_MAINTAINER:=John Marrett <johnf@zioncluster.ca>
+PKG_LICENSE:=
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/sscep
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+libopenssl +openssl-util
+  TITLE:=A Simple SCEP (Simple Certificate Enrollment Protocol) client
+  URL:=https://github.com/certnanny/sscep/
+endef
+
+define Package/sscep/description
+  sscep is a simple SCEP client published by the CertNanny project.
+
+  It can be used to retrieve CA certificates and enroll devices with a compatible SCEP server.
+endef
+
+define Build/Prepare
+$(call Build/Prepare/Default)
+	cd $(PKG_BUILD_DIR); ./bootstrap.sh
+endef
+
+define Build/Configure
+	cd $(PKG_BUILD_DIR); ./configure
+endef
+
+define Package/sscep/conffiles
+/etc/sscep.conf
+endef
+
+define Package/sscep/install
+	$(INSTALL_DIR) $(1)/etc
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/sscep.conf $(1)/etc/
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sscep $(1)/usr/bin/sscep
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mkrequest $(1)/usr/bin/
```

Comment:
```suggestion

	$(INSTALL_DIR) $(1)/usr/bin
	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sscep $(1)/usr/bin/sscep
	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mkrequest $(1)/usr/bin/mkrequest
```

Reply:
Sorry, this request isn't clear to me. I tried to switch from tabs to spaces as I thought that was what you were suggesting but it prevented the Makefile from working. I looked at the msgpack-c Makefile and it appears to follow the same tab and space usage as is in my Makefile.

Reply:
```patch
--- a/net/sscep/Makefile
+++ b/net/sscep/Makefile
@@ -37,6 +37,7 @@ endef
 define Package/sscep/install
 	$(INSTALL_DIR) $(1)/etc
 	$(INSTALL_CONF) $(PKG_BUILD_DIR)/sscep.conf $(1)/etc/
+
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sscep $(1)/usr/bin/sscep
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mkrequest $(1)/usr/bin/mkrequest
```